crates/wascap/src/jwt.rs

```diff
@@ -138,6 +138,14 @@ pub struct Invocation {
     pub invocation_hash: String,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Default)]
+pub struct Host {
+    /// Optional friendly descriptive name for the host
+    pub name: Option<String>,
+    /// Optional labels for the host
+    pub labels: Option<HashMap<String, String>>,
```

Comment:
Do you feel there's a meaningful difference between a `None` here and an empty hashmap? 

If we do change to an empty hashmap, it should be a `#[serde(default)]`, but this is probably a bikeshed

Reply:
I don't think there is, though I'd go a step further and ensure that it has a `#serde(skip_serializing_if = Option::is_none)]` which I should have done originally. If the host doesn't have labels then we skip serializing the key to begin with

Reply:
Updated to skip serializing if it's empty, so I think it should stay an `Option`